tests/pages/regions.py

```diff
@@ -0,0 +1,106 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Certainly good enough for now, however as this increases we may need a `regions` directory with suitable file names for the various regions. This will be useful for discovery.